session/pluginloader.go

```diff
@@ -112,13 +113,22 @@ func getUnixSocketDir(opts dgo.Map) string {
 	if v, ok := opts.Get("unixSocketDir").(dgo.String); ok {
 		return v.GoString()
 	}
-	if v := os.Getenv("TMPDIR"); v != "" {
+	if v := os.TempDir(); v != "" {
 		return v
 	}
 	return defaultUnixSocketDir
 }
 
-var defaultPluginTransport = "unix"
+// getDefaultPluginTransport returns the plugin transport method to use
+// for Windows this can be unix if the OS build is 17063 or above,
+// otherwise tcp is used
+// https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/
+func getDefaultPluginTransport() string {
+	if runtime.GOOS == "windows" && !isBuild17063() {
```

Comment:
There's still one problem. The `isBuild17063()` is undefined on all non-windows platforms. Even if it doesn't get called, there must be an implementation in a file with a `// +build !windows` directive.

I would suggest that you move the whole `getDefaultPluginTransport()` into one `// +build windows` and one `// +build !windows` file. In the windows version, the`runtime.GOOS == "windows"` becomes redundant and can be removed. The non-windows version can simply return "unix".

Reply:
Yes, I came to the same conclusion - looks like it's working now

Reply:
Excellent. Thanks!